guides/source/layouts_and_rendering.md

```diff
@@ -438,11 +438,11 @@ end
 private
 
 def determine_variant
-  variant = nil 
+  variant = nil
   # some code to determine the variant(s) to use
   variant = :mobile if session[:use_mobile]
-  
-  variant    
+
+  variant
```

Comment:
Why we even bother with the local `variant` variable here? Not needed as far as I can see.

Reply:
I didn't actually edit this example.  My text editor just trimmed the trailing whitespace on save.  I can edit it if desired, though.